src/masoniteorm/scopes/UUIDPrimaryKeyScope.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+from .BaseScope import BaseScope
+
+
+class UUIDPrimaryKeyScope(BaseScope):
```

Comment:
It's a little confusing but now that we have a global scope we register it to a mixin and then the mixin gets inherited by the model